### ISSUE TITLE
Icon Request 

### ISSUE THREAD

**Issue by firu82** [walkxcode/dashboard-icons#298]
### Type

Icon request

### Info

hello please add icon for mkvtoolnix,asustor nas and slskd.

Thanks

### Icon(s)

![mkvtoolnix](https://user-images.githubusercontent.com/26322641/222475508-e5ef0e56-d336-4b13-a93b-902cc1f375ed.png)

**Comment by walkxcode** [comment]
All issues are temporarily on hold. Please refer to #316 for more info.

**Comment by walkxcode** [comment]
Hey, this icon has been added in the `new-standards` branch and will be merged once it is done!

https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/mkvtoolnix.png
